coreos-base/oem-azure-pro/oem-azure-pro-2.2.46.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2013 CoreOS, Inc.. All rights reserved.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+DESCRIPTION="OEM suite for Azure"
+HOMEPAGE=""
+SRC_URI=""
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE=""
+
+# no source directory
+S="${WORKDIR}"
+
+RDEPEND="
```

Comment:
In the PR description you said that this duplicated the current oem-azure implementation, which could be refactored later... What I don't understand is why you didn't just add oem-azure as the RDEPEND here. Wouldn't that have accomplished the same result while avoiding the duplication?

Reply:
I think that would result in file conflicts for `base.ign`.

Reply:
Yes, due to the `base.ign` collision I decided to fork the `oem-azure`. I'm considering this a non-blocker, so my plan in the next releases is to look into refactoring this if possible.

Reply:
Alright, I see. Can you add a comment to this file stating that the duplication is due to the conflict of the base.ign file? Thanks!

Reply:
Sure, I've updated it with the comment.